community/libzip.hpkg

```diff
@@ -0,0 +1,66 @@
+(use ../prelude)
+(import ../core)
+(use ./cmake)
+(use ./zlib)
+
+(defsrc libzip-src
+  :file-name
+   "libzip-1-6-1.tar.gz"
+  :url
+   "https://github.com/nih-at/libzip/archive/rel-1-6-1.tar.gz"
+  :hash
+   "sha256:4f4e1991bf6b592e3d602d6b7b753c53ddadb2d76fca27d58120b8fadef7f1bf")
+
+(defpkg libzip
+  :builder
+   (fn []
+     (os/setenv "PKG_CONFIG_PATH"
+                (join-pkg-paths ":" "/lib/pkgconfig"
+                                 [zlib]))
+
+     (os/setenv "PATH"
+                (join-pkg-paths ":" "/bin"
+                                [core/awk
+                                 core/coreutils
+                                 core/pkgconf
+                                 core/gcc
+                                 core/grep
+                                 core/make
+                                 core/sed
+                                 cmake]))
+
+     (os/setenv "CFLAGS"
```

Comment:
I don't know why it works, but just using `*default-cflags*` seems to produce appropriate results.

Removing line 41 `"-L" (zlib :path) "/lib"` seems to succeed too, but not `(sh/$<_ pkg-config --libs zlib)` from line 49.

May be it's good to leave the lines in but possibly make comments that they are not necessary?

Reply:
I removed the `-L` and `-I` switches, because they (should) come from the invocation of `pkg-config`

Reply:
E.g.
```
PKG_CONFIG_PATH=/hpkg/59038568ddfdc0abe4b0143b20dfe952068df01a-zlib/lib/pkgconfig pkg-config --cflags zlib
=> -I/hpkg/59038568ddfdc0abe4b0143b20dfe952068df01a-zlib/include
```

Reply:
I've pulled from the branch and successfully built + run radare2 so it seems to be working here :)